kbase-extension/static/kbase/js/common/jobs.js

```diff
@@ -1,15 +1,25 @@
-define(['common/errorDisplay', 'common/format', 'common/html', 'common/props', 'common/ui'], (
+define(['common/errorDisplay', 'common/format', 'common/html', 'common/ui'], (
     ErrorDisplay,
     Format,
     html,
-    Props,
     UI
 ) => {
     'use strict';
 
     const t = html.tag,
         span = t('span');
 
+    const JOB = {
+        created: 'created',
+        estimating: 'estimating',
+        queued: 'queued',
+        running: 'running',
+        completed: 'completed',
+        error: 'error',
+        terminated: 'terminated',
+        does_not_exist: 'does_not_exist',
+    };
```

Comment:
because who wants to write out those annoying strings when there's a nice variable available?!